app/src/main/java/com/github/giommok/softwaredevproject/Database.java

```diff
@@ -42,6 +57,37 @@ public static void setChild(String path, List<String> childKeys, List<Object> va
         }
     }
 
+
+    /**
+     * Sets an array of values to the given child at the given path in the DB
+     * @param path The path to the fields to be modified
+     * @param child child key where to put the new values
+     * @param values list of values to add
+     */
+    public static void setChildObjectList(String path, String child,  ArrayList<Object> values){

```

Comment:
I don't think it's necessary to have both path and child parameters, as the child is part of the path.

Reply:
Changes made